docs/api/web-contents.md

```diff
@@ -650,6 +650,7 @@ Returns:
 * `params` Object
   * `x` Integer - x coordinate.
   * `y` Integer - y coordinate.
+  * `frame` WebFrameMain - Frame where the context menu was invoked in.
```

Comment:
nit: maybe something like "Frame from which the context menu was invoked"?

Reply:
Much nicer!